lib/ukrand/include/uk/swrand.h

```diff
@@ -0,0 +1,52 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef __UK_SWRAND__
+#define __UK_SWRAND__
+
+#include <sys/types.h>
+#include <uk/arch/types.h>
+#include <uk/plat/lcpu.h>
+#include <uk/config.h>
+#include <uk/plat/time.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define UK_SWRAND_CTOR_PRIO	1
+
+struct uk_swrand;
+
+extern struct uk_swrand uk_swrand_def;
+
+void uk_swrand_init_r(struct uk_swrand *r, unsigned int seedc,
+			const __u32 seedv[]);
+__u32 uk_swrand_randr_r(struct uk_swrand *r);
+
+__u32 uk_swrandr_gen_seed32(void);
```

Comment:
newline here... oh wait I see now, this is an older file also shown as added lines because you changed the license. Please do no overwrite previous licenses.